install/manifests/servicemonitor-lifecycle-app.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: monitor-project-lifecycle
+  labels:
+    # This tracks the default selector used by the stock prometheus-operator
+    # manifests.
+    k8s-app: monitor-project-lifecycle
+spec:
+  selector:
```

Comment:
can we modify the example to add namespaceSelector?

the more common scenario is that the target application to scrape will be in a different namespace than prometheus itself.

expect the monitored app would get deployed in a separate namespace than prometheus itself.

https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#servicemonitorspec

Reply:
We can, but for now I'm trying hard to just do everything in the `monitoring` namespace to remain compatible with the upstream manifests until we have our own which are more openshift centric (this is also why my app has a "k8s-app" label, for instance).